scripts/create-package/plop-templates-react/package.json.hbs

```diff
@@ -2,7 +2,6 @@
   "name": "{{{packageNpmName}}}",
   "version": "0.1.0",
   "description": "{{{description}}}",
-  "private": true,
```

Comment:
@ecraig12345 I updated the template to remove "private", so that new packages are published by default. Happy to revert this if it's not a good idea.